jormungandr/src/network/client/mod.rs

```diff
@@ -177,13 +177,14 @@ impl Client {
             // Ignoring possible Pending return here: due to the following
             // ready!() invocations, this function cannot return Continue
             // while no progress has been made.
-            block_sink.as_mut().poll_flush(cx).map_err(|e| {
-                error!(
+            match block_sink.as_mut().poll_flush(cx) {
```

Comment:
Looking at the above comment I think that it _might_ be a good idea just to disable this particular warning here.

Reply:
I'm down for it if you think it is clearer. But in the end I'm just taking the errors results here and discarding whatever else. Should be the same behavior as before right?